core/admin/mailu/internal/nginx.py

```diff
@@ -75,12 +76,16 @@ def get_status(protocol, status):
 
 
 def get_server(protocol, authenticated=False):
+    host_imap = os.environ["HOST_IMAP"] if "HOST_IMAP" in os.environ else "imap"
```

Comment:
In the admin interface, the configuration is declared in ``core/admin/mailu/__init__.py``, is overloaded by environment variables at startup and is then available through ``app.config``. I would prefer if you set a default value in the configuration declaration, that can be overloaded by the environment, then simply use ``app.config["HOST_IMAP"]`` and such where necessary.